src/engine/controls/cuecontrol.cpp

```diff
@@ -1787,6 +1995,60 @@ void CueControl::hotcueFocusColorNext(double v) {
     pCue->setColor(colorPalette.nextColor(*color));
 }
 
+void CueControl::setCurrentSavedLoop(CuePointer pCue) {
+    if (m_pCurrentSavedLoop != pCue) {
+        if (m_pCurrentSavedLoop) {
+            m_pCurrentSavedLoop->deactivate();
+        }
+
+        if (!pCue) {
+            m_pCurrentSavedLoop.reset();
+            return;
+        }
+
+        VERIFY_OR_DEBUG_ASSERT(pCue->getType() == mixxx::CueType::Loop &&
+                pCue->getEndPosition() != Cue::kNoPosition) {
+            return;
+        }
+
+        m_pCurrentSavedLoop = pCue;
+    }
+
+    if (m_pCurrentSavedLoop) {
+        qDebug() << "CueControl::setLoop" << pCue->getPosition() << pCue->getEndPosition() << true;
```

Comment:
What is the point of having a boolean literal in a debug message?
```suggestion
        qDebug() << "CueControl::setLoop" << pCue->getPosition() << pCue->getEndPosition();
```